### DISCUSSION TITLE
API KFA Integration

### DISCUSSION THREAD

**Discussion by ivanwilliammd** [ivanwilliammd/satusehat-integration#125] [Ideas]
Dear collaborator,
@YogiPristiawan @SyaefulKai @yudistirasd 

Need help untuk eksplorasi dan create new class dan feature untuk API KFA

https://satusehat.kemkes.go.id/platform/docs/id/master-data/kfa/rest-api-kfa/#apis-kfa-v2

Please kindly reply kalau ada yg bersedia untuk volunteer.
Thank you

**Comment by yudistirasd** [comment]
saya bisa di weekend ini.

**Comment by ivanwilliammd** [comment]
Sip thx @yudistirasd nanti bisa push ke branch https://github.com/ivanwilliammd/satusehat-integration/tree/31-api-kfa

**Comment by ivanwilliammd** [comment]
Note; @yudistirasd karena KFA ini masuk bagian dari terminology, jadi saya switch pathnya di commit terbaru ya

**Comment by yudistirasd** [comment]
Oh iya, siap pak.
Usul, untuk exception kalau dibuatkan folder khusus sendiri bagaimana pak? 
Sekalian buat exception untuk terminology

**Comment by ivanwilliammd** [comment]
Sip... sdh saya open branch di https://github.com/ivanwilliammd/satusehat-integration/issues/43 ya @yudistirasd
